js/assets.ts

```diff
@@ -1,51 +1,77 @@
 // Copyright 2018 Ryan Dahl <ry@tinyclouds.org>
 // All rights reserved. MIT License.
 
-// This file is formatted as it is because we are using the fact that Parcel
-// statically evaluates fs.readFileSync.
-import { readFileSync } from "fs";
+// tslint:disable-next-line:no-reference
+/// <reference path="plugins.d.ts" />
 
+// There is a rollup plugin that will inline any module ending with `!string`
```

Comment:
@ry -- this `import ... from "file!import_type"` is what we should support too IMO.
(not sure about the exclamation mark, but I believe `SystemJS` uses it too.)

Reply:
Yes, `SystemJS` uses load plugins that are post-pending after a `!`.  AMD uses prepending (e.g. `string!someMid.txt`).  I was going to use the AMD prepend, but TypeScript still throws a hissy fit with any MIDs that have extensions, even if they are pattern matched to the glob declaration, so I adopted the postpend.  While I don't know specifically where WHATWG is expecting to go, and if ES Module Loader plugins will ever be a standard thing, I would suspect they would adopt the SystemJS style.